<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-FQVHI6T" failures="4" tests="5" name="Test(failed)" time="50.053" errors="0" timestamp="2021-09-01T15:24:16 IST">
  <testcase classname="com.netBanking1.Testcases.TC_LoginTest_001" name="login" time="2.662"/>
  <testcase classname="com.netBanking1.Testcases.TC_LoginDDT_002" name="Login_Test" time="8.66">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.netBanking1.Testcases.TC_LoginDDT_002.Login_Test(TC_LoginDDT_002.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Login_Test -->
  <testcase classname="com.netBanking1.Testcases.TC_LoginDDT_002" name="Login_Test" time="6.122">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.netBanking1.Testcases.TC_LoginDDT_002.Login_Test(TC_LoginDDT_002.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Login_Test -->
  <testcase classname="com.netBanking1.Testcases.TC_LoginDDT_002" name="Login_Test" time="5.989">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.netBanking1.Testcases.TC_LoginDDT_002.Login_Test(TC_LoginDDT_002.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Login_Test -->
  <testcase classname="com.netBanking1.Testcases.TC_LoginDDT_002" name="Login_Test" time="7.562">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.netBanking1.Testcases.TC_LoginDDT_002.Login_Test(TC_LoginDDT_002.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Login_Test -->
</testsuite> <!-- Test(failed) -->
